src/styling/ArticlePage.css

```diff
@@ -6,6 +6,13 @@ article {
   font-size: 1.5rem;
 }
 
+@media (max-width: 800px) {
+  article {
```

Comment:
As a general rule you should use a class rather than applying component specific styling to all `article` tags. They're the only ones you have atm but keep it extensible by using classes.